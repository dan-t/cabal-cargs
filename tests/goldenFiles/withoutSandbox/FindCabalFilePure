lib exe tests -W -DLIB -DEXE dist/build/autogen dist/build/autogen cabal_macros.h