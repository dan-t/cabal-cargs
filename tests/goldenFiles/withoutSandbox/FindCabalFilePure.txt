lib exe tests -W -DLIB -DEXE base cmdargs lens directory strict transformers either text system-filepath system-fileio Cabal cabal-cargs tasty tasty-golden filepath dist/build/autogen dist/build/autogen cabal_macros.h .hdevtools.sock